From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Thu, 8 Aug 2025 00:00:00 -0800
Subject: [PATCH 2/2] Add MODULE.bazel file

Summary:
Adds MODULE.bazel file for bzlmod support

Test Plan:
Integration with cromatica project
---
 MODULE.bazel | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 MODULE.bazel

diff --git a/MODULE.bazel b/MODULE.bazel
new file mode 100644
index 0000000..0000000
--- /dev/null
+++ b/MODULE.bazel
@@ -0,0 +1,6 @@
+module(
+    name = "vulkan-headers",
+    version = "1.4.325",
+)
+
+bazel_dep(name = "rules_cc", version = "0.0.1")
-- 
2.34.1